diff --git a/node_modules/test-exclude/index.js b/node_modules/test-exclude/index.js
index aecb72d..74926b3 100644
--- a/node_modules/test-exclude/index.js
+++ b/node_modules/test-exclude/index.js
@@ -3,7 +3,8 @@
 const path = require('path');
 const { promisify } = require('util');
 const glob = promisify(require('glob'));
-const minimatch = require('minimatch');
+const mm = require('minimatch');
+const minimatch = typeof mm === 'function' ? mm : (mm.minimatch || mm.default);
 const { defaults } = require('@istanbuljs/schema');
 const isOutsideDir = require('./is-outside-dir');
 
